helmclient.go

```diff
@@ -721,9 +726,10 @@ func filterList(rels []*hapirelease.Release) []*hapirelease.Release {
 	return uniq
 }
 
-func getPod(client kubernetes.Interface, labelSelector, namespace string) (*corev1.Pod, error) {
+func getPod(client kubernetes.Interface, namespace string) (*corev1.Pod, error) {
 	o := metav1.ListOptions{
-		LabelSelector: labelSelector,
+		FieldSelector: tillerFieldSelector,
+		LabelSelector: tillerLabelSelector,
```

Comment:
We now also check that the tiller pod is running. We need to make a port forwarding connection so if its not running there is no point trying.